zowe-cli-id-federation-plugin/release.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    branches: [
```

Comment:
is this configuration correct?

Reply:
I guess no, because it's default one from cli-sample-plugin. We will be change it in follow-up PRs - to prepare plugin to release.

Reply:
I am ok to leave it here, just please make a comment that this is just "a template" or something

Reply:
I added comment in the beginner of the file.